controlplane/kubeadm/internal/workload_cluster.go

```diff
@@ -174,6 +179,39 @@ func (w *Workload) UpdateKubeletConfigMap(ctx context.Context, version semver.Ve
 		return err
 	}
 
+	// In order to avoid using two cgroup managers on the same machine,
+	// cgroupfs and systemd cgroup, starting from
+	// 1.21 image builder is going to configure containerd for using systemd cgroup,
+	// and the Kubelet configuration should be changed accordingly.
```

Comment:
```suggestion
	// In order to avoid using two cgroup managers on the same machine,
	// (cgroupfs and systemd cgroup drivers), starting from
	// 1.21 image builder is going to configure containerd for using the systemd driver,
	// and the Kubelet configuration must be changed accordingly.
```

these are "cgroup drivers"

Reply:
fixed